micropython/upysh/upysh.py

```diff
@@ -70,21 +82,42 @@ def newfile(path):
             f.write("\n")
 
 
-class Man:
+def rm(d):  # Remove file or tree
```

Comment:
This is recursive by default, like `rm -rf <path>`, which could be quite dangerous.  Is it worth having a `recurse=False` option to select recursion?

Reply:
I can add that option.